services/pom.xml

```diff
@@ -32,6 +32,8 @@
 
     <properties>
         <version.swagger.doclet>1.1.2</version.swagger.doclet>
+        <mockito.version>1.9.5</mockito.version>
```

Comment:
Please don't introduce new testing tools and rather write tests consistently with existing tests. See https://github.com/keycloak/keycloak/blob/master/CONTRIBUTING.md#testing.

Reply:
Ahh okay, I didn't know that. Will remove that again.

Reply:
Removed.